src/view/FrontPageView.java

```diff
@@ -16,11 +16,11 @@ public FrontPageView(FrontPageViewModel frontPageViewModel, SwitchViewController
         JButton appTitle = new ImageButton(FrontPageViewModel.TITLE_BUTTON_IMAGE_PATH, FrontPageViewModel.TITLE_BUTTON_IMAGE_SCALE);
         JButton searchButton = new ImageButton(FrontPageViewModel.SEARCH_BUTTON_IMAGE_PATH, FrontPageViewModel.SEARCH_BUTTON_IMAGE_SCALE);
 
+        appTitle.addActionListener(evt ->
+                openWebsiteController.execute(FrontPageViewModel.TITLE_BUTTON_REDIRECT_URL)
+        );
 
-
-        searchButton.addActionListener(evt -> {
-            switchViewController.execute(SearchViewModel.VIEW_NAME);
-        });
+        // TODO add listeners
```

Comment:
My codes are deleted again. There is definitely something wrong with how you handle merge conflicts.